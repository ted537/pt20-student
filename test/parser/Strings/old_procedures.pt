using input;
procedure s (var v: integer);
begin
    write (v); writeln;
end;
