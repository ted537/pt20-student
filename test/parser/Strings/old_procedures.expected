 .sProgram
   .sIdentifier
  % .sNewLine
  .sParmEnd
  .sBegin
     #eUnexpectedToken
